django/db/migrations/state.py

```diff
@@ -201,7 +201,7 @@ def register_model(self, app_label, model):
             self.app_configs[app_label] = AppConfigStub(app_label)
             self.app_configs[app_label].models = OrderedDict()
         self.app_configs[app_label].models[model._meta.model_name] = model
-        self.clear_cache()
+        self.clear_cache(True)
```

Comment:
In addition to my previous comment above, passing boolean flags as positional arguments should be avoided because the semantics is impossible to understand without checking the definition of the target function.